olad/Strerror_r.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Strerror_r.h
+ * Declaration of strerror_r that is XSI-compliant.
+ * Copyright (C) 2018 Shenghao Yang
+ */
+
+#ifndef OLAD_STRERROR_R_H_
+#define OLAD_STRERROR_R_H_
+
+namespace ola {
+
+/*
+ * @brief XSI-compliant version of @ref strerror_r()
+ 
+ * See strerror(3) for more details.
```

Comment:
@see

Reply:
`@see` is supposed to link to some symbol, though, AFAIK. Not sure what doxygen will do when it tries to link to some man page...

Reply:
It says "Starts a paragraph where one or more cross-references to classes, functions, methods, variables, files or URL may be specified." So an URL isn't a symbol, although I don't know how it decides if something is a URL or not?

Reply:
Hmm, I guess I could put a link to a manual page in man7.